plugins/imdb.py

```diff
@@ -5,7 +5,7 @@
 from cloudbot import hook
 
 id_re = re.compile(r'tt\d+')
-imdb_re = re.compile(r'(.*:)//(imdb.com|www.imdb.com)(:[0-9]+)?(.*)', re.I)
+imdb_re = re.compile(r'https?://(?:www\.)?imdb\.com[/\\]+title[/\\]+(tt[0-9]+)', re.I)
```

Comment:
Why do you match both `/` and `\` here?

Reply:
This is like a super edge edge case, but both are valid to use on IMDB: https://www.imdb.com\title\tt2575988\

Reply:
This is an invalid URL format, no browser I've tested will accept it.

Reply:
Tested via `echo -e "GET \\\title\\\tt25781\n\n" | nc imdb.com 80`, imdb.com returns a 503. Please remove this part of the pattern.